packages/core/test/wait.ts

```diff
@@ -0,0 +1,5 @@
+export function wait(durationMs: number = 0): Promise<void> {
```

Comment:
This is just generally a nice helper to have for async code; I'm using it in the updated async versions of the tests in `trackRuntimeError.spec.ts`.